src/Microsoft.AspNetCore.Localization/ApplicationBuilderExtensions.cs

```diff
@@ -43,12 +43,77 @@ public static IApplicationBuilder UseRequestLocalization(this IApplicationBuilde
             {
                 throw new ArgumentNullException(nameof(app));
             }
+            
             if (options == null)
             {
                 throw new ArgumentNullException(nameof(options));
             }
 
             return app.UseMiddleware<RequestLocalizationMiddleware>(Options.Create(options));
         }
+
+        /// <summary>
+        /// Adds the <see cref="RequestLocalizationMiddleware"/> to automatically set culture information for
+        /// requests based on information provided by the client. 
+        /// </summary>
+        /// <param name="app">The <see cref="IApplicationBuilder"/>.</param>
+        /// <param name="optionsAction"></param>
```

Comment:
This needs a description. It also needs to say that we're going to instantiate a **new** `RequestLocalizationOptions` that doesn't come from the services.